backend/app/adapter/sqldb/user_short_link.go

```diff
@@ -29,7 +29,7 @@ VALUES ($1,$2)
 		table.UserShortLink.ColumnShortLinkAlias,
 	)
 
-	_, err := u.db.Exec(statement, user.ID, shortLink.Alias)
+	_, err := u.db.Exec(statement, user.ID, shortLinkInput.GetCustomAlias(""))
```

Comment:
I wonder if I should instead pass the pointer `shortLinkInput.CustomAlias` directly here. It should never end up being `null` but if for some reason it is then the relationship should reflect that... 🤔

Reply:
I think it's safe and make sense to use `GetCustomAlias`. It's doesn't accept `nil`.

Reply:
After reading your other comments and reviewing the structure of the SQL tables, you're right. I will mark this as resolved.